body-parser should be in dependencies not devDependencies

### Version

[v4.8.1](https://github.com/auth-module/releases/tag/v4.8.1)

### Reproduction link

[http://url.com](http://url.com)

### Steps to reproduce

Try running a basic example without body-parser installed

```
Error while initializing nuxt: { /var/task/node_modules/@nuxtjs/auth/lib/providers/_utils.js:1
Error: Cannot find module 'body-parser'
```

### What is expected ?

It should work

### What is actually happening?

Nuxt can't find the body-parser module

<!--cmty--><!--cmty_prevent_hook-->
<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/auth-module/issues/c401">#c401</a>)</em></sub></div>

This is causing a problem when trying to deploy to ZEIT Now. Since `body-parser` is in `devDependencies`, the Now builder doesn't include it and therefore I get this error:

```
START RequestId: 2d8e5238-1967-4bb7-a6d8-6403abc144a0 Version: $LATEST
2019-09-02T18:59:21.306Z	2d8e5238-1967-4bb7-a6d8-6403abc144a0	λ Error while initializing nuxt: { /var/task/node_modules/@nuxtjs/auth/lib/providers/_utils.js:1
Error: Cannot find module 'body-parser'
Require stack:
- /var/task/node_modules/@nuxtjs/auth/lib/providers/_utils.js
- /var/task/node_modules/@nuxtjs/auth/lib/providers/github.js
- /var/task/node_modules/@nuxtjs/auth/lib/module/index.js
- /var/task/node_modules/@nuxt/core/dist/core.js
- /var/task/now__launcher.js
- /var/runtime/node_modules/awslambda/index.js
    at Object.<anonymous> (/var/task/node_modules/@nuxtjs/auth/lib/providers/_utils.js:2:20)
    at Object.Module._extensions..js (module.js:663:10)
  code: 'MODULE_NOT_FOUND',
  requireStack: 
   [ '/var/task/node_modules/@nuxtjs/auth/lib/providers/_utils.js',
     '/var/task/node_modules/@nuxtjs/auth/lib/providers/github.js',
     '/var/task/node_modules/@nuxtjs/auth/lib/module/index.js',
     '/var/task/node_modules/@nuxt/core/dist/core.js',
     '/var/task/now__launcher.js',
     '/var/runtime/node_modules/awslambda/index.js' ] }
END RequestId: 2d8e5238-1967-4bb7-a6d8-6403abc144a0
REPORT RequestId: 2d8e5238-1967-4bb7-a6d8-6403abc144a0	Duration: 308.28 ms	Billed Duration: 400 ms 	Memory Size: 3008 MB	Max Memory Used: 105 MB	
RequestId: 2d8e5238-1967-4bb7-a6d8-6403abc144a0 Process exited before completing request
```

This is blocking for anyone trying to deploy on ZEIT Now while using this module.

--- a/package.json
+++ b/package.json
@@ -42,6 +42,7 @@
   },
   "dependencies": {
     "@nuxtjs/axios": "^5.5.4",
+    "body-parser": "^1.19.0",
     "consola": "^2.9.0",
     "cookie": "^0.4.0",
     "dotprop": "^1.2.0",
@@ -54,7 +55,6 @@
     "@nuxtjs/toast": "^3.2.1",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.5",
-    "body-parser": "^1.19.0",
     "bootstrap-vue": "^2.0.0-rc.27",
     "codecov": "^3.5.0",
     "cookie-parser": "^1.4.4",

